### PR TITLE
HYDRATOR-1153 Fixing the documentation with JSON Parser Plugin

### PR DESCRIPTION
**Pull request by nitinmotgi**

### CODE REVIEW

File: transform-plugins/src/main/java/co/cask/hydrator/plugin/JSONParser.java [hunk 1]
```diff
@@ -167,11 +167,12 @@ public void transform(StructuredRecord input, Emitter<StructuredRecord> emitter)
...
   public static class Config extends PluginConfig {
     @Name("field")
-    @Description("Input Field")
+    @Description("Input field to be parsed as JSON")
     private String field;
 
     @Name("mapping")
-    @Description("Mapping complex JSON to fields using JSON Path expressions")
+    @Description("Maps complex JSON to output fields using JSON path expressions. First field defines the output " +
+      "field name and the second field specifies the JSON path expression.")
```
Comment (anew) [inline transform-plugins/src/main/java/co/cask/hydrator/plugin/JSONParser.java]: how about adding an example? :)

**Comment by abhinavbansal246** [comment]
LGTM
We should merge it in release/1.5

**Comment by anew** [comment]
I still think this should have en example in the documentation. It is very hard to understand without an example.

**Comment by anew** [comment]
LGTM

**Pull request merged**